frontend/src/components/chats/components/current-chat/components/message-form/common.ts

```diff
@@ -0,0 +1,12 @@
+import { ChatMessageCreateRequestBodyDto } from 'common/types/types';
+
+const getDefaultMessagePayload = (
```

Comment:
It would be better to have it in `frontend/src/components/chats/components/current-chat/components/message-form/helpers/helpers`

Reply:
I think, we need coaches advice here, because I do not see any reason to create a separate helper here

Reply:
in `common` folders we can't have the functions. you need to move it to the `helpers`